proposal (copy).md

```diff
@@ -0,0 +1,139 @@
+
+1. Consistency: Ensuring that every docstring has the same components
+   (parameters, return values, notes, examples, etc.). Ensuring that each
+   section is formatted correctly.
+2. Completeness: Ensuring that every function parameter is documented.
+3. Correctness: Ensuring that the examples run correctly.
+
+When a user submits a Pull Request to pandas, their changes to the documentation
+are automatically checked, and informative error messages inform them of any
+issues.
+
+Like many other projects, pandas uses the
+[numpydoc](https://numpydoc.readthedocs.io/en/latest/) standard for writing
+docstrings. With the collaboration of the numpydoc maintainers, we'd like to
+move the project-agnostic tooling we've written to a different project that any
+numpydoc-using project can benefit from.
+
+If possible, we'd like this project to be undertaken by a member of an
+unrepresented minority, with mentorship provided by the pandas maintaiers. This
+project primarily requires experience with *using* pandas, NumPy, and related
+libraries, rather that deep knowledge of pandas' internals. Pandas has a diverse
+community, just not at the maintainer level (yet).
+
+### Performance Monitoring
+
+Pandas uses [airspeed velocity](https://asv.readthedocs.io/en/stable/) to
+monitor for performance regressions. ASV itself is a fabulous tool, but requires
+some additional work to be integrated into an open source project's workflow.
+
+The [asv-runner](https://github.com/asv-runner) organization, currently made up
+of pandas maintainers, provides tools built on top of ASV. We have a physical
+machine for running a number of project's benchmarks and tools managing the
+benchmark runs and reporting on results.
+
+We'd like to fund improvements and maintenance of these tools to
+
+* Be more reliable. Currently, they're maintained on the nights and weekends
+  when a maintainer has free time. This occasionally results in days or weeks of
+  downtime.
+* Tune the system to improve benchmark stability
+* Report performance regressions to a project rather than relying on maintainers
+  manually checking for regressions.
+* Build a GitHub bot to request ASV runs *before* a Pull Request is merged. The
+  benchmarks are too expensive to run as part of every commit. Running on-demand
+  from a maintainer provides a nice balance.
+* Pay for hosting a dedicated benchmark server. A maintainer's basement isn't
+  the most stable environment for a machine.
+
+This project has an impact beyond pandas. We run the benchmarks for many
+foundational libraries in the Scientific Python Ecosystem including xarray,
+Dask, scikit-image, scikit-learn, and PyMC3. Each of these projects would
+benefit from improvements made to the tooling.
+
+## Existing Support
+
+Pandas currently has about two FTE.
+
+* [Anaconda Inc](https://www.anaconda.com) funds Tom Augspurger (50% FTE) and
+Brock Mendel (100% FTE)
+* [Ursa Labs](https://ursalabs.org) funds Joris van den Bossche (50% FTE).
```

Comment:
```suggestion
* [Ursa Labs](https://ursalabs.org) funds Joris van den Bossche (10% FTE).
```

is more correct, as the large part of my paid time is for arrow.

Reply:
Done in #4.